File: AbsEncap.cpp
#include<iostream>

using namespace std;

class AccessSpecifierDemo{
    public:
        int pubvar;
    private:
        int privar;
    protected:
        int provar;
    public:
        void setVar(int pubValue,int priValue,int proValue){
            pubvar= pubValue;
            privar= priValue;
            provar= proValue;
        }
        void getvar(){
            cout<<"The value of public variable : "<<pubvar<<endl;
            cout<<"The value of private variable : "<<privar<<endl;
            cout<<"The value of protected variable : "<<provar<<endl;
        };
};
int main(){
    AccessSpecifierDemo obj;
    int pri,pub,pro;
    cout<<"Enter the public,private,protected integer values : "<<endl;
    cin>>pub;
    cin>>pri;
    cin>>pro;
    obj.setVar(pub,pri,pro);  
    obj.getvar();
}

File: AbsEncap.java
import java.util.Scanner;

class AccessSpecifierDemo{
    int pubvar;
    protected 
        int provar;
    private
        int privar;
    
    void setVar( int pubValue,int priValue,int proValue)
    {
        pubvar = pubValue;
        privar = priValue;
        provar = proValue;
    }
    void getvar(){
        System.out.println("public variable : "+pubvar);
        System.out.println("private variable : "+privar);
        System.out.println("protected variable : "+provar);
        , int pubValue
    }
}
class AbsEncap{
    public static void main(String[] args){
        int pri,pro,pub;
        Scanner ip = new Scanner(System.in);
        AccessSpecifierDemo obj=new AccessSpecifierDemo();
        System.out.println("Enter the public,private,protected variables : ");
        pub = ip.nextInt();
        pri =ip.nextInt();
        pro = ip.nextInt();
        obj.setVar(pub,pri,pro);
        obj.getvar();
        ip.close();
    }
}
